### DISCUSSION TITLE
开源点评（2016-05-22） - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#98] [Announcements]
# 开源点评（2016-05-22） - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2016/2016_5/2016_5_22/

**Comment by ZoomQuiet** [comment]
> 是也乎(￣▽￣)

当年第二天的点评, 所有链接都正常,
可是一半打开已经非常艰难...
